src/main/scala/com/databricks/labs/overwatch/pipeline/Bronze.scala

```diff
@@ -95,6 +95,37 @@ class Bronze(_workspace: Workspace, _database: Database, _config: Config)
     Helpers.parClone(cloneSpecs)
 
   }
+  def snapshotStream(
+                      targetPrefix: String,
+                      excludes: Array[String] = Array()
+                    ): Unit = {
+    val bronzeTargets = getAllTargets :+ pipelineStateTarget
+
+    // if user provides dot path to table -- remove dot path and lower case the name
+    val cleanExcludes = excludes.map(_.toLowerCase).map(exclude => {
+      if (exclude.contains(".")) exclude.split("\\.").takeRight(1).head else exclude
+    })
+
+    // remove excludes
+    // remove non-existing bronze targets
+    val targetsToSnap = bronzeTargets
+      .filter(_.exists()) // source path must exist
+      .filterNot(t => cleanExcludes.contains(t.name.toLowerCase))
+
+    val finalTargetPathPrefix = s"${targetPrefix}/bckup"
```

Comment:
why are we nesting this into a subdir here?

Reply:
I think this makes sense if it were called something like `s"${targetPrefix}/data"` which would allow for another folder called `s"${targetPrefix}/report"` where we could persist the snapshot reports.